projects/gopacket/fuzz_layer.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Catena cyber All rights reserved.
```

Comment:
I don't think we can check in code with such copyright :/ Please use Google copyright or move this file to upstream repo (preferred)

Reply:
Ok, will try the preferred way